docs/en/ingest-management/tab-widgets/remove-endpoint-files/content.asciidoc

```diff
@@ -1,4 +1,7 @@
 // tag::mac[]
+
+//REVIEWERS: Are these commands still valid?
```

Comment:
@jmikell821 Should we have someone on the dev side confirm this?

I tried running the commands in macOS and Windows and they seem OK, but I needed to add `sudo` to the second line of macOS: `sudo cp /opt/Elastic/Endpoint/elastic-endpoint elastic-endpoint` (otherwise I got a permissions error). I don't have a Linux environment to check.